cli/internal/cmd/configgenerate.go

```diff
@@ -36,13 +37,27 @@ func newConfigGenerateCmd() *cobra.Command {
 	}
 	cmd.Flags().StringP("file", "f", constants.ConfigFilename, "path to output file, or '-' for stdout")
 	cmd.Flags().StringP("kubernetes", "k", semver.MajorMinor(config.Default().KubernetesVersion), "Kubernetes version to use in format MAJOR.MINOR")
+	cmd.Flags().StringP("attestation", "a", "", fmt.Sprintf("Attestation variant to use %s. If not specified, the default for the cloud provider is used.", printFormattedSlice(variant.GetAvailableAttestationTypes())))
 
 	return cmd
 }
 
+func printFormattedSlice[T any](input []T) string {
+	return fmt.Sprintf("{%s}", strings.Join(toString(input), "|"))
+}
+
+func toString[T any](t []T) []string {
+	var res []string
+	for _, v := range t {
+		res = append(res, fmt.Sprintf("%v", v))
+	}
+	return res
+}
```

Comment:
These should be moved down a bit based on: https://github.com/edgelesssys/wiki/blob/master/company/gostyle.md#order-of-elements-in-a-source-file

I would prefer `GetAvailableAttestationTypes` to be a `PrintAvailableAttestationTypes`. This would remove the need for these generic functions.

Reply:
I know YAGNI, but I thought its likely we will want to access all attestation types in the future. Whereas we would definitely not need the print function elsewhere